src/main/java/view/InputView.java

```diff
@@ -13,8 +20,32 @@ public static String getCarNames() {
         return scanner.nextLine();
     }
 
+    public static List<String> getSplitCarNames(final String names) {
+        List<String> splitNames = new ArrayList<>(Arrays.asList(names.split(",")));
+        duplicateCarNamesValidation(splitNames);
+        return splitNames;
+    }
+
+    private static void duplicateCarNamesValidation(List<String> splitNames) {
+        Set<String> distinctNames = new HashSet<>();
+        List<String> duplicatedNames = splitNames.stream()
+                .filter(name -> !distinctNames.add(name))
+                .toList();
+        if (!duplicatedNames.isEmpty()) {
```

Comment:
부정 (!) 예약어는 가독성을 떨어뜨린다는 의견이 있습니다.
저도 몇번 겪어본 바여서 메서드 추출을 통해 가독성 개선을 도모할때도 있습니다
```suggestion
        if (duplicatedNames.notEmpty()) {
```